docs/website/references/cli/defradb_client_action_list.md

```
@@ -0,0 +1,52 @@
+## defradb client action list
+
+List action information.
+
+### Synopsis
+
+List information pertaining to long running actions such as truncate, RefreshView,
+and the (re)building of indexes.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)